test/e2e/main_test.go

```diff
@@ -136,16 +140,21 @@ func TestMain(m *testing.M) {
 
 	cfg, _ := envconf.NewFromFlags()
 
-	clusterName := envconf.RandomName("crossplane-e2e", 32)
+	clusterName = envconf.RandomName("crossplane-e2e", 32)
 	if *kindClusterName != "" {
 		clusterName = *kindClusterName
 	}
 
 	// we want to create the cluster if it doesn't exist, but only if we're
 	isKindCluster := *create || *kindClusterName != ""
 	if isKindCluster {
+		kindCfg, err := filepath.Abs(filepath.Join("test", "e2e", "testdata", "kindConfig.yaml"))
+		if err != nil {
+			log.Log.Error(err, "error getting kind config file")
+			os.Exit(1)
+		}
 		setup = []env.Func{
-			envfuncs.CreateKindCluster(clusterName),
+			envfuncs.CreateKindClusterWithConfig(clusterName, "kindest/node:v1.27.3", kindCfg),
```

Comment:
This means we’ll have to remember to setup the kind version, we should setup renovate to update the version here as we are already doing in other places at least

Reply:
I set here the image version, only because there is no `CreateKind...` function that accepts only clusterName and config as arguments... if needed, we could create one, to avoid setting the image?

Reply:
This will result in kind using the default image, it's already what we actually rely on [here](https://github.com/upbound/crossplane/blob/7fd09b3f23cc5cd5cef438dbb80c05b3da3ea24b/cluster/local/kind.sh#L52), so I'd say it's fine to "hack" it here too, while we look for another option: 
```suggestion
			envfuncs.CreateKindClusterWithConfig(clusterName, "\"\"", kindCfg),
```
At least we won't have to update the version here too.

Reply:
I'll open an issue upstream to see if they directly handle the empty string properly in the meantime

Reply:
FYI a fix was merged and will be in the next release: https://github.com/kubernetes-sigs/e2e-framework/pull/297